src/xo-app/host/tab-patches.js

```diff
@@ -111,6 +114,18 @@ const INSTALLED_PATCH_COLUMNS_2 = [
   needsRestart: createDoesHostNeedRestart((_, props) => props.host)
 }))
 export default class HostPatches extends Component {
+  _installPatchWarning = ({patch, installPatch}) => confirm(
+    {
+      title: _('installPatchWarningTitle'),
+      body: <div><p>{_('installPatchWarningContent')}</p><Link to={'/pools/' + this.props.host.$pool + '/patches'} onClick={close}>link to the pool patches</Link></div>
+    }).then(() => installPatch(patch))
+
+  _installAllPatchesWarning = installAllPatches => confirm(
+    {
+      title: _('installPatchWarningTitle'),
+      body: <div><p>{_('installPatchWarningContent')}</p><Link to={'/pools/' + this.props.host.$pool + '/patches'} onClick={close}>link to the pool patches</Link></div>
+    }).then(() => installAllPatches())
```

Comment:
Use a centered green button with a text "Go to pool level patching". We want to be sure people won't miss it.